receiver/googlecloudmonitoringreceiver/config.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package googlecloudmonitoringreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudmonitoringreceiver"
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+const minCollectionIntervalSeconds = 60
+
+type Config struct {
+	scraperhelper.ControllerConfig `mapstructure:",squash"`
+
+	Region            string    `mapstructure:"region"`
+	ProjectID         string    `mapstructure:"project_id"`
+	ServiceAccountKey string    `mapstructure:"service_account_key"`
+	Services          []Service `mapstructure:"services"`
+}
+
+type Service struct {
+	ServiceName string        `mapstructure:"service_name"`
+	MetricName  string        `mapstructure:"metric_name"`
+	Delay       time.Duration `mapstructure:"delay"`
```

Comment:
This seems to duplicate functionality from `initial_delay` in scraperhelper.ControllerConfig.  Consider removing.

Reply:
We are using individual delay because certain services cannot display time series data immediately.
For exa- Read the **compute** service parameters [](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-compute)

Reply:
Does the list request return an error, or just no metrics? It might be easier to just not return metrics instead of requiring users to configure a delay.

Reply:
If the data is unavailable, a "no more items in iterator" error will be displayed.

To obtain time-series data based on delay configuration, we need to apply individual delays as specified. Otherwise, a global delay will suffice.

Reply:
sounds reasonable for now.